design/cluster-update.md

```diff
@@ -0,0 +1,209 @@
+# Cluster Updates
+
+## Background
+Currently, a Rook admin can declare how they want their cluster deployed by specifying values in the [Cluster CRD](../Documentation/cluster-crd.md).
+However, after a cluster has been initially declared and deployed, it is not currently possible to update the Cluster CRD and have those desired changes reflected in the actual cluster state.
+This document will describe a design for how cluster updating can be implemented, along with considerations, trade-offs, and a suggested scope of work.
+
+## Design Overview
+As previously mentioned, the interface for a user who wants to update their cluster will be the Cluster CRD.
+To specify changes to a Rook cluster, the user could run a command like the following:
```

Comment:
Is this section with editing the Cluster CRD overall obsolete with the removal of the `versionTag`?

But how will updates now be triggered? Just by changing the operator image version?

Reply:
The user experience of editing the Cluster CRD via `kubectl` (or the k8s API) is still very relevant without `versionTag`, I expect that to be the main way a user will update their cluster over time.

Software version upgrades will be triggered by the operator when a user has updated the operator of the version itself, so that clusters always match the version of the operator.